Lib/string.py

```diff
@@ -146,6 +142,7 @@ def convert(mo):
                              self.pattern)
         return self.pattern.sub(convert, self.template)
 
+Template.__init_subclass__()  # setup pattern
```

Comment:
This is a bit of a head scratcher, but I think the comment could be improved to make it more clear.  What's really going on is that this is required to initialize `pattern` when no subclass is created.  Can you improve the clarity of the comment (and maybe move it above this line, and make it a full sentence)?   If you need a suggestion, let me know.

Reply:
I would appreciate your help with a comment.